test/parallel/test-mime-api.js

```diff
@@ -1,8 +1,9 @@
+// Flags: --expose-internals
 'use strict';
 
 require('../common');
 const assert = require('assert');
-const { MIMEType, MIMEParams } = require('util');
+const { MIMEType, MIMEParams, toASCIILower } = require('internal/mime');
```

Comment:
I think we still want to test the user exposed API (in case we introduce a different one internally)
```suggestion
const { MIMEType, MIMEParams } = require('util');
const { toASCIILower } = require('internal/mime');
```